Setup Dev Environment/Readme.md

```diff
@@ -0,0 +1,28 @@
+# Setup-development-environment
+The Repository is created to track the internship activities at ITIC
+
+
+TO DO :
+
+-Setup Python 3.6 or 3.7 on your machine.
```

Comment:
How? Add commands e.g. the following:
```
brew install --ignore-dependencies https://raw.githubusercontent.com/Homebrew/homebrew-core/f2a764ef944b1080be64bd88dca9a1d80130c558/Formula/python.rb
```

Reply:
I installed python using the python executable file on my windows machine. I have updated the steps in  the document explaining how to install python on the machine